sormas-api/src/main/java/de/symeda/sormas/api/person/PersonDto.java

```diff
@@ -119,6 +122,13 @@
 	@PersonalData(mandatoryField = true)
 	@SensitiveData(mandatoryField = true)
 	private String lastName;
+	@HideForCountriesExcept
+	@PersonalData
+	@SensitiveData
+	private Salutation salutation;
+	@PersonalData
+	@SensitiveData
+	private String otherSalutation;
```

Comment:
What is the intention of property `otherSalutation`? If possible, rename this to clarify which context this property is intended to be used in instead of `salutation` (is it a `customSalutation`?)
If the intention is to allow salutations not covered by the enum, it might make sense to add an additional method here to resolve the salutation, something like
```java
return otherSalutation != null ? otherSalutation : salutation.toString();
```

Reply:
@syntakker it is a `customSalutation` used when the salutation id OTHER.

I don't see the use of the additional method you described and we don't have that kind of methods

Reply:
@leventegal-she the conditional statement from my comment is needed whenever a String is rendered from a person's salutation, so I would advocate to implement it in a central place (like it is done in `LocationDto.checkIsEmptyLocation`).